app/move/views/_includes/court-information.njk

```diff
@@ -0,0 +1,66 @@
+<section class="app-!-position-relative">
+  {% if move.from_location.location_type == 'prison' and not courtSummary.rows.length %}
```

Comment:
Why has the conditional moved inside the `section` element?

Reply:
The original conditional has been moved to the `view.njk` template. However, there is still another condition that needs to remain to choose whether to display court hearings or court information as they aren't shown for all move types.